src/pages/invitee/EventInvite.jsx

```diff
@@ -94,17 +128,18 @@ const EventInvite = () => {
               <input
                 name="preferred_date_time"
                 type="datetime-local"
-                placeholder="17/11/2022 - 3pm"
```

Comment:
- [ ] KIndly note that the format at which this placeholder date is structured is the date required by the backend, so make sure you are sending your date in this format for consistency purpose